modules/util/portal-tools-rest-builder/src/main/resources/com/liferay/portal/tools/rest/builder/dependencies/dto.ftl

```diff
@@ -139,6 +141,10 @@ public class ${schemaName} <#if dtoParentClassName?has_content>extends ${dtoPare
 			sizeParameters = []
 		/>
 
+		<#if !stringUtil.equals(propertyName, "properties")>
```

Comment:
What is the purpose of this empty if clause? :thinking:

Reply:
It was a old if statement that I used before some changes, I forgot deleting it sorry.